### PR TITLE
NodeJS 16 image has reached EOL in April 2024

### PR DESCRIPTION
**Pull request by phracek**
NodeJS 16 image has reached EOL in April 2024 as for RHEL8 as for RHEL9.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,6 @@ Images available on Quay are:
 * CentOS 7 [nodejs-12](https://quay.io/repository/centos7/nodejs-12-centos7)
 * CentOS 7 [nodejs-14](https://quay.io/repository/centos7/nodejs-14-centos7)
 * Fedora [nodejs-14](https://quay.io/repository/fedora/nodejs-14)
```
Comment (zmiklank) [inline README.md:9]: Please remove also CentOS 7 images and Fedora nodejs-14 from list of supported images in this README.

**Review by zmiklank — COMMENTED** [review]
Just a nitpick.

File: README.md (hunk 1 above)
Comment (phracek) [reply README.md]: Done by commit https://github.com/sclorg/s2i-nodejs-container/pull/430/commits/2b9ff57c4f43b990f04dff01f6fc4a7c3619600d

**Review by zmiklank — APPROVED** [review]
Thanks, LGTM.

**Pull request merged**